docs/madr/decisions/056-meshexternalservice-and-zoneegress.md

```diff
@@ -0,0 +1,407 @@
+# MeshExternalService and ZoneEgress
+
+* Status: accepted
+
+Technical Story: https://github.com/kumahq/kuma/issues/10897
+
+## Context and Problem Statement
+
+MeshExternalService traffic can go out directly from the sidecar or through egress.
+
+Configuration (like timeouts, retries, circuit breakers) of MeshExternalService can also be applied both on a sidecar and on an egress.
+
+This MADR will go into the details of:
+- limitations / features that occur / are possible when running the traffic directly from sidecar vs through egress
+- each policy type and the placement to figure out what is the best option
+
+## Decision Drivers
+
+* not restricting any useful use-cases
+* flexible enough that we don't limit users in scenarios where it makes sense to configuration in both/one place
```

Comment:
nit: "not restricting useful use-cases" and "don't limit users in scenarios" sound like the same decision driver :)

Reply:
removed one